plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/OpenShiftImages.java

```diff
@@ -49,6 +49,8 @@
 	public static final Image SERVICE_IMG = repo.getImage("service.png"); //$NON-NLS-1$ 
 	public static final ImageDescriptor ROUTE = repo.create("route.png"); //$NON-NLS-1$
 	public static final Image ROUTE_IMG = repo.getImage("route.png"); //$NON-NLS-1$
+	public static final ImageDescriptor PROJECT_NEW = repo.create("project.png"); //$NON-NLS-1$
```

Comment:
"project.png" descriptor being already initialized, these 2 lines cause an initialization error in the explorer.
I removed them